projects/index.html

```diff
@@ -1,11 +1,11 @@
 ---
-title: Projects
-excerpt: "This is where we keep a collection of some of the projects that people are working on using ev3dev. We invite you to click through the links below to see what cool stuff ev3dev can do!"
+title: Project Showcase
+excerpt: "This is where we keep a collection of some of the most interesting projects that people are working on using ev3dev. We invite you to click through the links below to see what cool stuff ev3dev can do!"
```

Comment:
When I bought the mindstorms set (for my kids, of course), I've created all of the stock robots from that page, and programmed those in python. I still have the sources, so I could update those to the latest kernel version if we decide to do this.